test/evergreen.yml

```diff
@@ -590,18 +593,14 @@ functions:
         script: |
           set -o errexit
           set -o verbose
-          # Parse through any failures inside the WT_TEST directory.
-          ${python_binary|python3} ../test/evergreen/print_stack_trace.py --unit_test -e ${python_binary|python3} -c WT_TEST -l .
+          ${python_binary|python3} ../test/evergreen/print_stack_trace.py
 
-          # Look through the stress format directory for any coredumps.
-          ${python_binary|python3} ../test/evergreen/print_stack_trace.py --format -e ./test/format/t -c test/format/. -l test/format/.
     - command: s3.put
       params:
         aws_secret: ${aws_secret}
         aws_key: ${aws_key}
         local_files_include_filter:
           - wiredtiger/cmake_build/*stacktrace.txt
```

Comment:
@Siddhartha8899, does this look into:
```
cmake_build/test/*/*stacktrace.txt
```

Reply:
Why do we have to look into that folder? I though we run it from the cmake_build folder and dump everything there, no?

Reply:
You are right, my bad. Resolving.